Admin/Admin.php

```diff
@@ -1306,6 +1306,31 @@ public function hasRoute($name)
 
         return $this->routeGenerator->hasAdminRoute($this, $name);
     }
+    
+    /**
+     * {@inheritdoc}
```

Comment:
Inherit doc from what? :wink: 
Could you please provide a real documentation docblock, so that everybody can understand why «developers need this method» ?